File: CSTH/OLD/test2/pack1/Edge.java
package pack1;

public class Edge{
  public int id1,id2;
  private double length;
  public Edge(int verA,int verB,double len){
    id1 = verA;
    id2 = verB;
    length = len;
  }
  public int getVerA(){
    return id1;
  }
  public int getVerB(){
    return id2;
  }
  public double getLength(){
    return length;
  }
}

File: CSTH/pack1/Edge.py
class Edge:
	def __init__(self, verA, verB, length) -> None:
		self.id1 = verA
		self.id2 = verB
		self.length = length
	
	def getVerA(self):
		return self.id1

	def getVerB(self):
		return self.id2
	
	def getLength(self):
		return self.length
	